src/reporters/lang/en.js

```diff
@@ -132,6 +132,8 @@ const messages = {
   linkUnregistered: 'Unregistered $0.',
   linkUsing: 'Using linked module for $0.',
 
+  createUnsupportedScope: 'Scoped packages cannot be installed via yarn create for now - read <github issue> to know why.',
```

Comment:
Instead of adding an error here, I'd probably just support it out of the box. We discussed 

```
yarn create @foo/react-app
```

would translate to: `@foo/yarn-create-react-app`. Can we implement it this way?